tempesta_fw/gfsm.c

```diff
@@ -260,11 +265,28 @@ tfw_gfsm_move(TfwGState *st, unsigned short state, const TfwFsmData *data)
 			r = TFW_POSTPONE;
```

Comment:
Not regarding this PR, but losely related with GFSM question: if one of HTTP FSM hooks returned `TFW_POSTPONE`
and e.g. in `tfw_http_req_process()`:
https://github.com/tempesta-tech/tempesta/blob/97184f60a78b90ed0703b8c4c4edcefd8290c794/tempesta_fw/http.c#L2681
we get `data_off == skb_len`, then `tfw_http_req_process()` will return `TFW_POSTPONE` too and lower-level processing will close connection. Is this valid behavior?

Reply:
No, but this is covered in #1012